draft-meunier-http-message-signatures-directory.md

```diff
@@ -99,10 +99,16 @@ defined in {{configuration}}.
 ## Header Field Definition
 
 The `Signature-Agent` header field is an Item Structured Header {{STRUCTURED-HEADERS}}. Its value MUST be a
-String containing a {{URI}}. The ABNF is:
+Dictionary which member values are {{URI}}. The ABNF is:
```

Comment:
On the lien above you state the field is an item, which its not when t's a Dictionary :smile:

Reply:
good catch. fixed